libraries/AP_Motors/AP_Motors_Class.h

```diff
@@ -98,6 +99,7 @@ class AP_Motors {
     void                set_throttle_filter_cutoff(float filt_hz) { _throttle_filter.set_cutoff_frequency(filt_hz); }
     void                set_forward(float forward_in) { _forward_in = forward_in; }; // range -1 ~ +1
     void                set_lateral(float lateral_in) { _lateral_in = lateral_in; };     // range -1 ~ +1
+    virtual void        set_roll_pitch(float roll_deg, float pitch_deg) {};
```

Comment:
I think it will be easy for future devs to get confused between the use of "set_roll_pitch" and the individual "set_roll" and "set_pitch" calls so let's try and unify them or clarify the difference in the comments at least.